helm-charts/verticadb-operator/values.yaml

```diff
@@ -18,7 +18,14 @@
 
 
 image:
-  name: vertica/verticadb-operator:1.3.1
+  repo: null # null = dockerhub
+  name: vertica/verticadb-operator
+  tag: 1.3.1
+
+rbac_proxy_image:
```

Comment:
I'm okay with adding a config knob for the rbac_proxy_image.  Is there a distinct advantage in splitting the image name into 3 parts?  I prefer to minimize the number of helm chart parameters and it complicates the chart a bit.

Reply:
There is an advantage to splitting the repo name from everything else: it allows users to override the hostname (e.g. to a mirror, air gaped server, etc) without changing package name and version. This make it easier to upgrade operator versions in the future without changing chart configuration.

Re the tag: I just did that for symmetry. It's a pattern I saw in other charts, e.g. [csi-driver-smb/values.yaml](https://github.com/kubernetes-csi/csi-driver-smb/blob/master/charts/v1.5.0/csi-driver-smb/values.yaml).

Do you have a preferred config structure? I'm happy to implement it.

Reply:
Thanks for the additional background.  Let's stick with just splitting the repo name for now and not have a separate tag parm.  That should cleanup the e2e tests -- the `deploy-operator` target in the Makefile does a `helm install` and it currently doesn't account for the tag parm.